go.sum

```diff
@@ -607,6 +607,14 @@ github.com/libp2p/go-openssl v0.1.0 h1:LBkKEcUv6vtZIQLVTegAil8jbNpJErQ9AnT+bWV+O
 github.com/libp2p/go-openssl v0.1.0/go.mod h1:OiOxwPpL3n4xlenjx2h7AwSGaFSC/KZvf6gNdOBQMtc=
 github.com/livepeer/ai-worker v0.12.1 h1:V6XGxnRmq02GuJP/PYRKTXIZAd2F1jTuErweO5boWxU=
 github.com/livepeer/ai-worker v0.12.1/go.mod h1:/Deme7XXRP4BiYXt/j694Ygw+dh8rWJdikJsKY64sjE=
+github.com/livepeer/ai-worker v0.12.2-0.20241115144717-cffea497e22e h1:/rGRpiHAhaCzk8fV+RXjkVILea8wSoqpuDQY/CdpilY=
+github.com/livepeer/ai-worker v0.12.2-0.20241115144717-cffea497e22e/go.mod h1:/Deme7XXRP4BiYXt/j694Ygw+dh8rWJdikJsKY64sjE=
+github.com/livepeer/ai-worker v0.12.2-0.20241115145337-b8477a4881ae h1:suFeJeJqm+PKrtJJO7ubkb9m+6CZPOHbO0xQmxycUEI=
+github.com/livepeer/ai-worker v0.12.2-0.20241115145337-b8477a4881ae/go.mod h1:/Deme7XXRP4BiYXt/j694Ygw+dh8rWJdikJsKY64sjE=
+github.com/livepeer/ai-worker v0.12.2-0.20241115145705-4cefa2dc7463 h1:k81OkHOR4EBPEBcM8DhFUmkdiL4BdMyZMgwmJ2N/0+E=
+github.com/livepeer/ai-worker v0.12.2-0.20241115145705-4cefa2dc7463/go.mod h1:/Deme7XXRP4BiYXt/j694Ygw+dh8rWJdikJsKY64sjE=
+github.com/livepeer/ai-worker v0.12.2-0.20241118111050-0a9cd07bca51 h1:+mKaLa8DvxxBJd9KTpbfN48fse+34HteqavOpCklDlI=
+github.com/livepeer/ai-worker v0.12.2-0.20241118111050-0a9cd07bca51/go.mod h1:/Deme7XXRP4BiYXt/j694Ygw+dh8rWJdikJsKY64sjE=
```

Comment:
go mod tidy?

Reply:
Update in ac9b327ce52369c1e89ba7a01e319e6d471f57a0